concent_api/api-e2e-additional-verification-test.py

```diff
@@ -426,6 +430,8 @@ def test_case_6_test_without_script_src_in(cluster_consts: ProtocolConstants, cl
         from concent_api.settings import STORAGE_CLUSTER_ADDRESS
         from concent_api.settings import ADDITIONAL_VERIFICATION_TIME_MULTIPLIER
         from concent_api.settings import BLENDER_THREADS
+        cluster_name = get_cluster_name_from_address(sys.argv[1])
```

Comment:
Why not doing this directly in `SCIBaseTest`?

Reply:
Moved to `SCIBaseTest`

Reply:
I see it still there, have you pushed those changes?

Reply:
Yes, I have pushed all of these changes and now I checked it again and it is in `sci_testing_common.py`